keps/sig-apps/4040-controller-name/kep.yaml

```diff
@@ -0,0 +1,35 @@
+title: Controller name
+kep-number: 4040
+authors:
+  - "@mwielgus"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-autoscaling
+status: implementable
+creation-date: 2023-05-29
+reviewers:
+  - "@alculquicondor"
+approvers:
+  - "@soltysh"
+  - "@gjtempleton"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
```

Comment:
Dropping this KEP with literal days left before the KEP freeze is unfortunate.  I didn't get to read the PRR sections and I have significant concerns about the approach.